dropwizard-jdbi/src/main/java/io/dropwizard/jdbi/DBIFactory.java

```diff
@@ -28,18 +31,36 @@
     private static class SanerNamingStrategy extends DelegatingStatementNameStrategy {
         private SanerNamingStrategy() {
             super(NameStrategies.CHECK_EMPTY,
-                  NameStrategies.CONTEXT_CLASS,
-                  NameStrategies.CONTEXT_NAME,
-                  NameStrategies.SQL_OBJECT,
-                  new StatementNameStrategy() {
-                      @Override
-                      public String getStatementName(StatementContext statementContext) {
-                          return RAW_SQL;
-                      }
-                  });
+                    NameStrategies.CONTEXT_CLASS,
```

Comment:
I realize it's a little late now, but formatting changes like this (particularly pure whitespace changes) that have no impact typically aren't something you want to include in a PR unless you are rectifying a significant style deviation. The reason being that now it becomes more difficult to determine the original source of the change for these lines.